src/store/mutations.js

```diff
@@ -232,6 +241,10 @@ export default {
 			list.splice(idx, 1)
 		}
 
+		if (!envelope.seen && mailbox.unread) {
+			Vue.set(mailbox, 'unread', mailbox.unread - 1)
```

Comment:
in an edge case `removeEnvelope` is run twice concurrently and then this *might* result in a negative number. Same if there is a sync request right before this code is run.

We can leave it like this but some hardening in a follow-up PR wouldn't hurt :)